### ISSUE TITLE
🚀 Feature: Onboarding and role selection for new comers, adding company onboarding flow 

### ISSUE THREAD

**Issue by VineeTagarwaL-code** [code100x/job-board#520]
### 🔖 Feature description

![image](https://github.com/user-attachments/assets/f74d8088-a03b-4c7a-a0ee-63261db4fbb9)

### 🎤 Why is this feature needed ?

To allow HR's to onboard easily too 

### ✌️ How do you aim to achieve this?

Implemented the flowchart given above 

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/code100x/job-board/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by DeepanshuMishraa** [comment]
Assign me

**Comment by Dhirajsingh212** [comment]
Assign

**Comment by VineeTagarwaL-code** [comment]
what will be the eta for this ? whoever does this first will be merged

**Comment by DeepanshuMishraa** [comment]
> what will be the eta for this ? whoever does this first will be merged

On it ser

**Comment by CuriousCoder00** [comment]
> what will be the eta for this ? whoever does this first will be merged

@VineeTagarwaL-code for it to be done #496 needs to be merged first
![image](https://github.com/user-attachments/assets/532077b6-c7ad-4ed6-ba85-7f762ac33ee4)

**Comment by VineeTagarwaL-code** [comment]
@CuriousCoder00 we have hr the role already

**Comment by CuriousCoder00** [comment]
yeah, we have HR role but in the mentioned pr the job model and company model was separated 
before company details and job details were in a single model
made changes in job model and created a new model company and then made relation in b/w

**Comment by VineeTagarwaL-code** [comment]
@DeepanshuMishraa the schema right now looks something like this 

![image](https://github.com/user-attachments/assets/82f70d4c-d90e-43a2-9f5f-5905d1d512b2)


the companies details is present inside the job schema, seperate them and add the company as a 1:1 relation to the job entity

**Comment by CuriousCoder00** [comment]
@VineeTagarwaL-code I've already done the separation part... 
![image](https://github.com/user-attachments/assets/41348f5b-d58d-47e2-8baf-c10c5ca061d2)

**Comment by CuriousCoder00** [comment]
just need to do a few testings

**Comment by DeepanshuMishraa** [comment]
> @DeepanshuMishraa the schema right now looks something like this
> 
> ![image](https://private-user-images.githubusercontent.com/91052168/378169900-82f70d4c-d90e-43a2-9f5f-5905d1d512b2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0MTQ5OTksIm5iZiI6MTcyOTQxNDY5OSwicGF0aCI6Ii85MTA1MjE2OC8zNzgxNjk5MDAtODJmNzBkNGMtZDkwZS00M2EyLTlmNWYtNTkwNWQxZDUxMmIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIwVDA4NTgxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0Njg4MWViYzc4MzMzYmMxY2FlNWQxNjExODg0MGY1ZjUwN2M1Yzc5MDYxYTZiNmVjMzg4MGRmYzA4YTY5NWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QQhV3chghOEOz07kOa1j_A34SsnFZTbGb2vASQlmQ_8)
> 
> the companies details is present inside the job schema, seperate them and add the company as a 1:1 relation to the job entity

Yeah will do it

**Comment by VineeTagarwaL-code** [comment]
> just need to do a few testings
> For #496 

Okay on it, will update shortly
